backend/lib/utils.js

```diff
@@ -46,21 +59,18 @@ const lookupVersion = async(platform, bundleId, country = 'us') => {
       throw e;
     }
 
-    res = res.data;
+    const resData = res.data;
+
+    // Version
+    const version = getTokenValue("Current Version", "Requires", resData)
+
+    // Release Date
+    const released = dateFns.parse(getTokenValue("Updated", "Size", resData), 'LLLL d, yyyy', new Date());
```

Comment:
Could we add some try-catching around here so that if `getTokenValue` returns something that isn't a valid date, this doesn't prevent the rest of the output from being sent to the client?

Reply:
Sure.

I actually have more changes that I've made to make the android version info to be more consistent with the iOS one.

Will include that in.

Reply:
@tschoffelen

I updated the code to fallback to a default value if release date or notes are unavailable, is this sufficient? or would you like try catch for each retrieved value?